src/client/src/core/util/validation/rules.ts

```diff
@@ -627,3 +627,12 @@ export const validLangString = function(value: string) {
     ? translate('general.invalid')
     : true;
 };
+
+/**
+ * Validate autocomplete selection
+ * @param {string|object|null} value
+ * @returns {boolean|string}
+ */
+export const validSelection = function(value: string | object | null) {
```

Comment:
```suggestion
export const validAutocompleteSelection = function(value: string | object | null) {
```
:thinking:

Reply:
Seems a bit verbose. Considering theres no other rules for selections